tests/models/onnxModels/model.onnxtxt

```diff
@@ -0,0 +1,133 @@
+ir_version: 3
```

Comment:
curious, how do you generate this serialized proto?

Reply:
I wrote a python file to generate the proto. I could upload the two files -- one is for caffe2 model generating and one is for onnx model generating in our repo, and later we could modify those files to generate the models we want.

Reply:
yes, would be great to have those utility scripts in the repo checked-in.